<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRPlot</class>
 <widget class="QMainWindow" name="QRPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1307</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QRPlot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="plottedProperties">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="propertyInfo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_graph">
       <attribute name="title">
        <string>Graph</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCustomPlot" name="plot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>plot</zorder>
       <zorder>plot</zorder>
       <zorder>plot</zorder>
      </widget>
      <widget class="QWidget" name="tab_table">
       <attribute name="title">
        <string>Table</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1318</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionExportGraphics"/>
     <addaction name="actionExportDataCSV"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuExport"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_file"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open Summary File</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExportGraphics">
   <property name="text">
    <string>Graphics</string>
   </property>
   <property name="toolTip">
    <string>Export image as png</string>
   </property>
  </action>
  <action name="actionExportDataCSV">
   <property name="text">
    <string>Data as CSV</string>
   </property>
   <property name="toolTip">
    <string>Export the displayed data in a CSV format</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/icons/folder.png</normaloff>:/toolbar/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="toolTip">
    <string>Open RSM summary file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
